app/controllers/manage_cmr/permissions_controller.rb

```diff
@@ -242,7 +246,7 @@ def get_groups
     filters = {
       'provider'  => current_user.provider_id,
       :page_num  => 1,
-      :page_size => 50
+      :page_size => 1000000
```

Comment:
Do we need to consider needing to paginate here? I'm not sure if we can really retrieve that many records using CMR but, there may also be sufficiently less groups than the MAX page size

Reply:
This page size is a big misleading, as it is mimicking some backgrounds capability functionality that used to page.   The paging is actually now only used to process a chunk of results returned in order to fetch the # of users.   This is just telling to to process the entire results.